src/modules/launcher/PowerLauncher/ReportWindow.xaml

```diff
@@ -5,19 +5,70 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              mc:Ignorable="d" 
              WindowStartupLocation="CenterScreen"
+             Background="{DynamicResource SystemChromeLow}"
              Icon="Images/app_error.dark.png"
              Topmost="True"
              ResizeMode="NoResize"
-             Width="600"
-             Height="455"
+             Width="760"
+             Height="560"
              Title="{DynamicResource reportWindow_wox_got_an_error}"
              d:DesignHeight="300" d:DesignWidth="600" x:ClassModifier="internal">
-    <RichTextBox x:Name="ErrorTextbox" 
-                 IsDocumentEnabled="True"
-                 VerticalScrollBarVisibility="Auto" 
-                 HorizontalScrollBarVisibility="Auto" 
-                 FontSize="14" 
-                 Margin="10"
-                 BorderThickness="0"/>
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="pack://application:,,,/Styles/ModernScrollBarStyle.xaml" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+    <Grid Margin="24">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="64"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+        
+        <TextBlock Text="Something went wrong."
```

Comment:
Hi @niels9001. Great work.

Two things I want to mention:
1. Text suggestion: "Oops! Something went wrong. :("
2. Should we move the text for translation into a resource file?